### PR TITLE
Dilshod changes

### PR DESCRIPTION
**Pull request by dilshod707**
Hi first Pull request

### CODE REVIEW

File: src/test/java/tests/ClickAccount.java [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
 
-public class ProfileAndAccount {
+public class ClickAccount extends TestBase {
 
+    @Test
     public void ProfileAndAccount() {
 
       new LoginTests().positiveLogin();
-        ProfilePage profilePage = new ProfilePage();
-        ProfilePage
```
Comment (sdetkamala) [inline src/test/java/tests/ClickAccount.java]: you should delete this

File: src/test/java/tests/ClickAccount.java [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
 
+import org.testng.annotations.Test;
+import pages.Account;
 import pages.LoginPage;
 import pages.ProfilePage;
 import utils.Driver;
 
 import java.time.Duration;
 
-public class ProfileAndAccount {
```
Comment (sdetkamala) [inline src/test/java/tests/ClickAccount.java]: this too

File: src/test/java/tests/ClickAccount.java [hunk 3]
```diff
@@ -1,19 +1,26 @@
...
+public class ClickAccount extends TestBase {
 
+    @Test
     public void ProfileAndAccount() {
 
       new LoginTests().positiveLogin();
-        ProfilePage profilePage = new ProfilePage();
-        ProfilePage
         Driver.getDriver().manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
-        profilePage.getAccountLink().click();
```
Comment (sdetkamala) [inline src/test/java/tests/ClickAccount.java]: this one too

Comment (dilshod707) [reply src/test/java/tests/ClickAccount.java]: Thanks for feedback, i will send you corrected ones

**Pull request closed without merging**